### PR TITLE
New UnitTest from Alex to 7

### PR DESCRIPTION
**Pull request by dragkb**
Please review my PR

### CODE REVIEW

File: app/src/test/java/com/example/Vorobiov/ExampleUnitTest.java [hunk 1]
```diff
@@ -52,5 +52,20 @@ public void testFizzBuzz5(){
...
+        expectedRes.add("2");
+        expectedRes.add("Fizz");
+        expectedRes.add("4");
+        expectedRes.add("Buzz");
+        expectedRes.add("Fizz");
+        expectedRes.add("7");
+        List<String> actualResult = algo.fizzBuzz(7);
+
+        Assert.assertArrayEquals("Expected variance from Actual", actualResult.toArray(), expectedRes.toArray());
+        System.out.println("I want to be SDET!!!");
```
Comment (Serhii-Vorobiov) [inline app/src/test/java/com/example/Vorobiov/ExampleUnitTest.java]: What's not necessary!

Comment (dragkb) [reply app/src/test/java/com/example/Vorobiov/ExampleUnitTest.java]: What do you mean?

Comment (Serhii-Vorobiov) [reply app/src/test/java/com/example/Vorobiov/ExampleUnitTest.java]: I meant that you should delete 69 line.

Comment (dragkb) [reply app/src/test/java/com/example/Vorobiov/ExampleUnitTest.java]: You don't like the comment or sout method ?

Comment (Serhii-Vorobiov) [reply app/src/test/java/com/example/Vorobiov/ExampleUnitTest.java]: We do not need this method in our project=) So please delete it!

**Comment by dragkb** [comment]
I deleted the method you asked for.

**Pull request merged**